phpstan.neon

```diff
@@ -0,0 +1,9 @@
+parameters:
+  level: 1
+  paths:
+    - ./src
+  ignoreErrors:
+    - '#Class [a-zA-Z0-9\\_:\(\).]* extends [a-zA-Z0-9\\_:@\(\).]*#'
```

Comment:
Why were these error ignored? @narendradesai

Reply:
@saxumVermes These are class not found errors. `Class <class_name> extends unknown class <class_name> `. These errors are thrown because classes are present outside scope of the repository.

Reply:
This is a really good example why this should not be done. Please next time include the reason in the solution! I've just read why they tagged the Client as final. the reason being, when version 8 comes out the will set the class (not just annotating) as final. They wanted to mark it as a deprecated that way, which will break backward compatibility with all our implementations essentially disabling - without refactor - updating to an upper version.
This needs to be taken care of soon.